addons/go.mod

```diff
@@ -4,25 +4,23 @@ go 1.16
 
 replace (
 	github.com/vmware-tanzu/tanzu-framework => ../
-	k8s.io/api => k8s.io/api v0.19.2
-	k8s.io/client-go => k8s.io/client-go v0.19.2
-	sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v0.3.11-0.20201204161359-8437691189ad
+	sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.0.0
```

Comment:
Replace cluster-api could be removed as the version is the same.

Reply:
I think this is required. Please refer to https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/2467

Reply:
yes, that is required. Let me add a comment on that line.